tests/utils.go

```diff
@@ -538,17 +538,17 @@ func CleanNodes() {
 
 		old, err := json.Marshal(node)
 		Expect(err).ToNot(HaveOccurred())
-		new := node.DeepCopy()
+		newNode := node.DeepCopy()
 
 		k8sClient := GetK8sCmdClient()
 		if k8sClient == "oc" {
-			RunCommandWithNS("", k8sClient, "adm", "uncordon", node.Name)
+			_, _, _ = RunCommandWithNS("", k8sClient, "adm", "uncordon", node.Name)
```

Comment:
I would benefit if you split this commit and add a word about why this particular change is helpful.

IMHO this misses the most important bit: you don't collect `err` and do not do anything if it has occurred.

Reply:
Will split.

This indeed doesn't change the fact that we're ignoring a potential error, but it makes it obvious.
In this case, it's probably fine to ignore it as we're just cleaning up as much as we can at the end of a test run.

What I'm trying to do with this PR is to reduce the number of existing warnings (in IDEs like goland) as much as possible.
I was modifying this file for another PR, and couldn't easily ensure I didn't introduce any new warning because there were so many already...